docs/timeline/index.html

```diff
@@ -2239,10 +2239,26 @@ <h3 id="available-locales">Available locales</h3>
       <td>Portuguese</td>
       <td>
         <code>pt</code><br>
-        <code>pt_BR</code>
+        <code>pt_BR</code><br>
```

Comment:
This merge request should only touch the Swedish and Norwegian parts and leave the other code alone for not. It Portuguese need change this should be done in a seperate merge request.